<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="resetPushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gameGraphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="seedSpinBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="targetSpinBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="playPushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="seedValueLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seed value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="targetValueLabel1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Target value: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="closePushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="targetValueLabel2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(as a power of 2)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrowUpPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrowRightPushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrowDownPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrowLeftPushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="currentScoreTextBrowser">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="highscoreTextBrowser">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pointsLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your points:</string>
    </property>
   </widget>
   <widget class="QLabel" name="highScoreLabel1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your highscore:</string>
    </property>
   </widget>
   <widget class="QLabel" name="highScoreLabel2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(this session)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minLcdNumber">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="secLcdNumber">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="minLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="secLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pausePushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="targetValueTextBrowser">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="equalsLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>493</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu2048">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionPause"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionSuomi"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInstructions"/>
   </widget>
   <addaction name="menu2048"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInstructions">
   <property name="text">
    <string>Instructions</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSuomi">
   <property name="text">
    <string>Suomi</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
